src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -5073,40 +5068,33 @@ public override void Accept(OperationVisitor visitor)
     /// </summary>
     internal sealed partial class UsingStatement : BaseUsingStatement, IUsingStatement
     {
-        public UsingStatement(IOperation body, IVariableDeclarationStatement declaration, IOperation value, SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit) :
+        public UsingStatement(IOperation expression, IOperation body, SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit) :
```

Comment:
>expression [](start = 41, length = 10)

This isn't necessary an expression, consider renaming.

Reply:
Fixed.